src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -210,7 +210,8 @@ void uct_rc_mlx5_iface_common_prepost_recvs(uct_rc_mlx5_iface_common_t *iface)
 {
     /* prepost recvs only if quota available (recvs were not preposted
      * before) */
-    if (iface->super.rx.srq.quota == 0) {
+    if ((iface->super.rx.srq.quota == 0) ||
+        (iface->rx.srq.type == UCT_IB_MLX5_OBJ_TYPE_NULL)) {
```

Comment:
why the check iface->super.rx.srq.quota == 0 is not enough? can we set quota to 0 for gga?